pkg/karmadactl/taint.go

```diff
@@ -173,22 +153,21 @@ func (o *CommandTaintOption) Validate() error {
 
 // AddFlags adds flags to the specified FlagSet.
 func (o *CommandTaintOption) AddFlags(flags *pflag.FlagSet) {
-	o.GlobalCommandOptions.AddFlags(flags)
-
 	flags.BoolVar(&o.overwrite, "overwrite", o.overwrite, "If true, allow taints to be overwritten, otherwise reject taint updates that overwrite existing taints.")
 	flags.BoolVar(&o.DryRun, "dry-run", false, "Run the command in dry-run mode, without making any server requests.")
+	flags.StringVar(defaultConfigFlags.KubeConfig, "kubeconfig", *defaultConfigFlags.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
```

Comment:
As @jwcesign comment on https://github.com/karmada-io/karmada/pull/2611/files 
Can we extract these duplicated codes ?  @carlory